src/jats/importer/jats-body-transformations.ts

```diff
@@ -194,19 +207,34 @@ export const jatsBodyTransformations = {
     references: BibliographyItem[] | null,
     createElement: (tagName: string) => HTMLElement
   ) {
+    const bodyContainer = createSectionContainer('body', createElement)
+    const abstractsContainer = createSectionContainer(
+      'abstracts',
+      createElement
+    )
+    const backmatterContainer = createSectionContainer(
+      'backmatter',
+      createElement
+    )
+
+    const bodySections = doc.querySelectorAll('body > sec')
+    for (const section of bodySections) {
+      removeNodeFromParent(section)
+      bodyContainer.append(section)
+    }
```

Comment:
I would prefer to separate this logic of wrapping all sections with our own special section to a new method. and we will call it after `moveSectionsToBody`